tests/integration/test_apply_checks.py

```diff
@@ -804,19 +1583,47 @@ def test_apply_checks_with_is_unique(ws, spark, set_utc_timezone):
             [
```

Comment:
In test_apply_checks_with_is_unique, missing EXTRA_PARAMS when initializing DQEngine so that times won't match in the results.